gc/scheduler/scheduler.go

```diff
@@ -306,8 +310,8 @@ func (s *gcScheduler) run(ctx context.Context) {
 		}
 
 		s.waiterL.Lock()
-
-		stats, err := s.c.GarbageCollect(ctx)
+		tracCtx, startSpan := tracing.StartSpan(ctx, makeSpanName("StartGarbageCollect"))
```

Comment:
StartGarbageCollect -> GarbageCollect? It seems the span is for an entire garbage collection.

Reply:
`StartGarbageCollect` change to `gc.schedule.Run`.